packages/gtk3.rb

```diff
@@ -3,25 +3,23 @@
 class Gtk3 < Package
   description 'GTK+ is a multi-platform toolkit for creating graphical user interfaces.'
   homepage 'https://developer.gnome.org/gtk3/3.0/'
-  @_ver = '3.24.29'
+  @_ver = '3.24.30'
   @_ver_prelastdot = @_ver.rpartition('.')[0]
   version @_ver
   license 'LGPL-2.1'
-  compatibility 'all'
+  compatibility 'x86_64 aarch64 armv7l'
   source_url 'https://gitlab.gnome.org/GNOME/gtk.git'
   git_hashtag @_ver
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk3/3.24.29_armv7l/gtk3-3.24.29-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk3/3.24.29_armv7l/gtk3-3.24.29-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk3/3.24.29_i686/gtk3-3.24.29-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gtk3/3.24.29_x86_64/gtk3-3.24.29-chromeos-x86_64.tpxz'
+    aarch64: 'file:///usr/local/tmp/packages/gtk3-3.24.30-chromeos-armv7l.tpxz',
+     armv7l: 'file:///usr/local/tmp/packages/gtk3-3.24.30-chromeos-armv7l.tpxz',
+     x86_64: 'file:///usr/local/tmp/packages/gtk3-3.24.30-chromeos-x86_64.tpxz'
```

Comment:
Not gitlab binaries?

Reply:
Ugh. Didn't do a last `git add .` before that last push. Fixed.